### PR TITLE
Process Kill Charts

### PR DESCRIPTION
**Pull request by neelanjan00**
## Why we need this PR
- Adds process-kill experiment charts

### CODE REVIEW

File: charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
       - apiGroups: ["litmuschaos.io"]
         resources: ["chaosengines","chaosexperiments","chaosresults"]
         verbs: ["create","list","get","patch","update","delete"]
-    image: "litmuschaos/go-runner:ci"
+    image: "litmuschaos/go-runner:latest"
```
Comment (uditgaurav) [inline charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml:45]: If it's on `m-agent` branch let's give him the m-agent tag only.

File: charts/os/process-kill/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    resources: ["secrets","configmaps"]
+    verbs: ["get","list",]
+  # Track and get the runner, experiment, and helper pods log 
+  - apiGroups: [""]
+    resources: ["pods/log"]
+    verbs: ["get","list","watch"]  
+  # for creating and managing to execute comands inside target container
+  - apiGroups: [""]
+    resources: ["pods/exec"]
+    verbs: ["get","list","create"]
```
Comment (uditgaurav) [inline charts/os/process-kill/rbac.yaml]: Do we need this? Also, the comments are misleading.

File: charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml (hunk 1 above)
Comment (neelanjan00) [reply charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml:45]: This is the aws-ssm experiment chart, not the process-kill experiment chart. The process-kill experiment image has been tagged with `m-agent` tag only.

File: charts/os/process-kill/rbac.yaml (hunk 2 above)
Comment (neelanjan00) [reply charts/os/process-kill/rbac.yaml]: Removed the pods/exec resources from RBAC and experiment manifests.

**Review by uditgaurav — APPROVED** [review]

File: charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml (hunk 1 above)
Comment (uditgaurav) [reply charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml:45]: I see

**Pull request merged**